src/qspi/Adafruit_FlashTransport_QSPI_SAMD.cpp

```diff
@@ -80,6 +80,12 @@ void Adafruit_FlashTransport_QSPI::begin(void) {
   QSPI->CTRLA.bit.ENABLE = 1;
 }
 
+void Adafruit_FlashTransport_QSPI::end(void) {
+  QSPI->CTRLA.bit.ENABLE = 0;
+
+  /* NEEDS WORK */
```

Comment:
We should also disable the qspi clock as well (reversed to being() ) to save power. 

```C
MCLK->APBCMASK.bit.QSPI_ = false;
MCLK->AHBMASK.bit.QSPI_ = false;
MCLK->AHBMASK.bit.QSPI_2X_ = false;
```

Reply:
Done in https://github.com/adafruit/Adafruit_SPIFlash/pull/108/commits/a40a06418d01698968022b08f45d7cec3bf184b7